cdk/kraken/src/deploy.ts

```diff
@@ -60,21 +74,14 @@ export class DeployJob extends CheckoutJob {
           },
         },
         {
-          name: 'Deploy',
-          run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
-
-          # get repo name from synth step
-          RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}
-
-          # Deploy
-          kubectl apply -f k8s/dist/ -l app.kubernetes.io/component=certificate
-          kubectl apply -f k8s/dist/ --prune -l app.kubernetes.io/part-of=$RELEASE_NAME`,
-          env: {
-            AWS_ACCOUNT_ID: '${{ secrets.AWS_ACCOUNT_ID }}',
-            AWS_ACCESS_KEY_ID: '${{ secrets.GH_AWS_ACCESS_KEY_ID }}',
-            AWS_SECRET_ACCESS_KEY: '${{ secrets.GH_AWS_SECRET_ACCESS_KEY }}',
-          },
-        },
+          name: 'Push to kube-manifests repository',
```

Comment:
We can push manifests to another repo, but we should still keep & run the deployment steps:

```
kubectl apply -f k8s/dist/ -l app.kubernetes.io/component=certificate
kubectl apply -f k8s/dist/ --prune -l app.kubernetes.io/part-of=$RELEASE_NAME`
```

Or did you plan for this to be ran as a part of the github actions for the `kube-manifests` repository? Might make more sense to keep deployments specific to products imo and just push to `kube-manifests` for record-keeping. 

Also, can you update the snapshots?

Reply:
The idea behind this PR is migrating to use argocd to manage our deployments rather than manually kubectl applying to the cluster. We can push yaml files to the `kube-manifests` repo, which argo is monitoring and it can then update our cluster using those changes.

Reply:
I've added the eks deploy step back in for now. It's likely easier to deploy to kube-manifests and kubectl deploying while the Argo deployment is getting ironed out and then removing the kubectl deploy step afterwards.

Reply:
Using both argo and the direct kubectl apply feels like we may end up with some weird race condition. It may make sense to remove the kubectl apply here,, finish the kittyhawk rollout, then migrate over to argo.